### DISCUSSION TITLE
Cell Temperature in Charging Stats

### DISCUSSION THREAD

**Discussion by baldohunter** [CarlosCuezva/dashboards-Grafana-Teslamate#31] [Ideas]
I don't know enough about Json and Grafana to do it but could you add the average / min / max Cell temperature to the Charging details? That would be nice in Winter time.

something like this:

{
„matcher“: {
„id“: „byRegexp“,
„options“: „^cellavg_temp_.*“
},
„properties“: [
{
„id“: „displayName“,
„value“: „AVG Cell Temperature“
}
]
},

**Comment by CarlosCuezva** [comment]
Hi baldohunter:

Which dashboard specifically are you referring to?

**Comment by baldohunter** [comment]
As an example the one attached from jheredianet/Teslamate-CustomGrafanaDashboards

[Charge Details-1703017560519.json](https://github.com/CarlosCuezva/dashboards-Grafana-Teslamate/files/13720054/Charge.Details-1703017560519.json)


https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards


That one is almost perfect, except min max Avg cell temp.

And in the same style for slow charging voltage of every single cell.

**Comment by CarlosCuezva** [comment]
ok, but I still don't understand in which panel you want that feature

**Comment by baldohunter** [comment]
Hi
I attached two pictures one is original second is with a solid red line Avg
temp ans dotted Line min/max cell temp.

Regards
Thomas

Carlos Cuezva ***@***.***> schrieb am Fr., 22. Dez. 2023,
15:29:

**Reply by CarlosCuezva** [reply]
No attached image appears

**Comment by baldohunter** [comment]
Shit could you give me your direct email? The mail is something from github

Carlos Cuezva ***@***.***> schrieb am Fr., 22. Dez. 2023,
15:39:

**Comment by baldohunter** [comment]
Second try 
![Screenshot_20231222_153457_Samsung Internet](https://github.com/CarlosCuezva/dashboards-Grafana-Teslamate/assets/154230389/7740a3aa-f802-4580-8d0f-4ca2162c23df)
![Screenshot_20231222_153407_Samsung Internet](https://github.com/CarlosCuezva/dashboards-Grafana-Teslamate/assets/154230389/726bad27-b205-4af0-a233-b12754955004)
